### DISCUSSION TITLE
如何在插件模块中引用其他单文件py模块

### DISCUSSION THREAD

**Discussion by NatsumiXD** [nonebot/discussions#181] [问答]
如何在插件模块中引用其他单文件py模块？

**Comment by yanyongyu** [comment]
直接import就行啊，具体条件可以看https://v2.nonebot.dev/docs/next/tutorial/plugin/load-plugin#%E7%A1%AE%E4%BF%9D%E6%8F%92%E4%BB%B6%E5%8A%A0%E8%BD%BD%E5%92%8C%E8%B7%A8%E6%8F%92%E4%BB%B6%E8%AE%BF%E9%97%AE

**Reply by NatsumiXD** [reply]
要以文件夹的形式加载插件？

**Reply by NatsumiXD** [reply]
我直接引用他说无法加载（

**Reply by yanyongyu** [reply]
如果报错说插件在加载前被import，那你就需要优先声明你依赖的插件或者使用require，上面给的链接里有说明

**Reply by NatsumiXD** [reply]
非常感谢

**Reply by NatsumiXD** [reply]
问题来了，重载时显示：
ModuleNotFoundError: No module named 'fenshu'

**Reply by NatsumiXD** [reply]
而fenshu.py就存在于插件同目录下，为何会加载失败呢？

**Reply by yanyongyu** [reply]
你的nb版本是最新版吗

```python
from .fenshu import xxxx
from . import fenshu
```

**Reply by NatsumiXD** [reply]
好像不是

**Reply by NatsumiXD** [reply]
是最新的了，使用nb-cli创建

**Reply by yanyongyu** [reply]
你把你的目录结构和bot.py截个图
